sass/components/nav.sass

```diff
@@ -131,10 +131,22 @@ a.nav-item
   // Modifiers
   &.has-shadow
     box-shadow: 0 2px 3px rgba($black, 0.1)
+  &.is-fixed-top
+    position: fixed
+    left: 0
+    right: 0
+    top: 0
+    +mobile
+      position: static
   // Responsiveness
   +touch
     & > .container,
     .container > &
       & > .nav-left
         & > .nav-item.is-brand:first-child
           padding-left: 20px
+
+.has-fixed-nav
+  padding-top: $nav-height
+  +mobile
+    padding-top: 0
```

Comment:
Due to the nav bar returning to a non-sticky state on mobile, padding-top is not required.